balboa-agent/src/test/scala/com/socrata/balboa/agent/MetricConsumerSpec.scala

```diff
@@ -238,6 +278,23 @@ class MetricConsumerSpec extends WordSpec with ShouldMatchers with MockitoSugar
             fileQueues(rootMetricsDir).size - 1)
         }
       }
+
+      "it successfully reads in all the metrics from the file" should {
+        "emit all of them" in new OneRootDirectory with MetadataTestMetrics {
+          def slurp(resourceName: String) = {
+            val resource = getClass.getClassLoader.getResource(resourceName)
+            val file = resource.getFile
+            val bis = new BufferedInputStream(new FileInputStream(file))
+            try Stream.continually(bis.read).takeWhile(-1 !=).map(_.toByte).toArray
```

Comment:
These two lines are difficult to read.  Can you add `{}` around the `try` and `finally` blocks, and maybe break up the `Stream` line a bit?